x/mint/spec/02_state.md

```diff
@@ -8,28 +8,36 @@ order: 2
 
 The minter is a space for holding current inflation information.
 
- - Minter: `0x00 -> amino(minter)`
-
-```go
-type Minter struct {
-	Inflation        sdk.Dec   // current annual inflation rate
-	AnnualProvisions sdk.Dec   // current annual exptected provisions
+ - Minter: `0x00 -> ProtocolBuffer(minter)`
+
+```protobuf
+message Minter {
```

Comment:
we could actually use `+++`, see https://github.com/cosmos/cosmos-sdk/pull/8269#discussion_r553886663